src/target/riscv/riscv.c

```diff
@@ -4501,6 +4583,27 @@ static const struct command_registration riscv_exec_command_handlers[] = {
 		.help = "Execute a sequence of 32-bit instructions using the program buffer. "
 			"The final ebreak instruction is added automatically, if needed."
 	},
+	{
+		.name = "equality_match_trigger_enable",
+		.handler = riscv_equality_match_trigger_enable,
+		.mode = COMMAND_CONFIG,
+		.usage = "[on|off]",
+		.help = "Allow OpenOCD to use equality match trigger in bp/wp"
+	},
+	{
+		.name = "napot_trigger_enable",
+		.handler = riscv_napot_trigger_enable,
+		.mode = COMMAND_CONFIG,
+		.usage = "[on|off]",
+		.help = "Allow OpenOCD to use NAPOT trigger in bp/wp"
+	},
+	{
+		.name = "ge_lt_trigger_enable",
+		.handler = riscv_ge_lt_trigger_enable,
+		.mode = COMMAND_CONFIG,
+		.usage = "[on|off]",
+		.help = "Allow OpenOCD to use GE/LT triggers in bp/wp"
+	},
```

Comment:
When commands are added or changed, the manual of OpenOCD (`doc/openocd.texi`) should be updated, too.

Reply:
Addressed